zstd_vendor/package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
```

Comment:
3 -> 2

Reply:
These probably flew under the radar since I used `ros2 pkg create` and didn't pay attention to the default version.